bok_choy/web_app_test.py

```diff
@@ -34,6 +36,57 @@ def __init__(self, *args, **kwargs):
         self.har_mode = getattr(
             self, 'har_mode', os.environ.get('BOK_CHOY_HAR_MODE', None))
 
+    @classmethod
+    def setUpClass(cls):
+        """
+        Override NeedleTestCase's setUpClass method so that it does not
+        start up the browser once for each testcase class.
+        Instead we start up the browser once per TestCase instance,
+        in the setUp method.
+        """
+        if os.environ.get('NEEDLE_CAPTURE'):
+            cls.capture = True
+        if os.environ.get('NEEDLE_SAVE_BASELINE'):
+            cls.save_baseline = True
```

Comment:
This is the mechanism that needle uses to create the screenshots. I'm writing some doc and will include a warning.